### ISSUE TITLE
missing libintl.h 

### ISSUE THREAD

**Issue by johnr2017** [espressif/esp-idf#2969]
----------------------------- Delete below -----------------------------

If your issue is a general question, starts similar to "How do I..", or is related to 3rd party development kits/libs, please discuss this on our community forum at esp32.com instead.

INSTRUCTIONS
============

Before submitting a new issue, please follow the checklist and try to find the answer.

- [x] I have read the documentation [ESP-IDF Programming Guide](https://docs.espressif.com/projects/esp-idf/en/latest/) and the issue is not addressed there.
- [x] I have updated my IDF branch (master or release) to the latest version and checked that the issue is present there.
- [x] I have searched the issue tracker for a similar issue and not found a similar issue.

If the issue cannot be solved after the steps before, please follow these instructions so we can get the needed information to help you in a quick and effective fashion.

1. Fill in all the fields under **Environment** marked with [ ] by picking the correct option for you in each case and deleting the others.
2. Describe your problem.
3. Include [debug logs on the monitor](https://docs.espressif.com/projects/esp-idf/en/latest/get-started/idf-monitor.html#automatically-decoding-addresses) or the [coredump](https://docs.espressif.com/projects/esp-idf/en/latest/api-guides/core_dump.html).
4. Provide more items under **Other items if possible** can help us better locate your problem.
5. Use markup (buttons above) and the Preview tab to check what the issue will look like.
6. Delete these instructions from the above to the below marker lines before submitting this issue.

----------------------------- Delete above -----------------------------

## Environment

- Development Kit:      [|ESP32-DevKitC|- 

- - IDF version (run ``git describe --tags`` to find it): 
  
  v3.2-dev-1148-g96cd3b75c
- Build System:         [Make|CMake]
- Compiler version 
(crosstool-NG crosstool-ng-1.22.0-80-g6c4433a5) 5.2. 0

- Operating System:   Windows

## Problem Description
Fresh install of complete toolchain. Followed instructions from Espressif site

I loaded the IDF, Toolchain and new Gcc compilers. 
**There  is a problem with a  missing file: libintl.h**
(I could not find it either)
latest installer and toolchain: msys2-i686-20161025.exe.

This is from the BASH window when I type "make menuconfig" in the hello_world example
HOW CAN I FIX THIS PROBLEM??

John@DESKTOP-1FHIRUI MINGW32 ~/esp/hello_world                                                                     $ make menuconfig 

make[1]: Entering directory `/c/Users/John/esp/esp  idf/tools/kconfig'                                              /usr/bin/gcc -c  -DCURSES_LOC="" -DLOCALE -MD  /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c -o mconf. o                                                                                                                  In file included from /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c:22:                                          /c/Users/John/esp/esp-idf/tools/kconfig/lkc.h:12:22: libintl.h: No such file or directory                          In file included from /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c:23:                                          /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:38:20: curses.h: No such file or directory               In file included from /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c:23:                                          /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:103: error: parse error before "chtype"                  /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:103: warning: no semicolon at end of struct or union     /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:107: error: parse error before '}' token                 /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:117: error: field `screen' has incomplete type           /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:118: error: field `shadow' has incomplete type           /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:119: error: field `dialog' has incomplete type           /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:120: error: field `title' has incomplete type            /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:121: error: field `border' has incomplete type           /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:122: error: field `button_active' has incomplete type    /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:123: error: field `button_inactive' has incomplete type  /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:124: error: field `button_key_active' has incomplete typ e                                                                                                                  /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:125: error: field `button_key_inactive' has incomplete t ype                                                                                                                /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:126: error: field `button_label_active' has incomplete t ype                                                                                                                /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:127: error: field `button_label_inactive' has incomplete  type                                                                                                              /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:128: error: field `inputbox' has incomplete type         /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:129: error: field `inputbox_border' has incomplete type  /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:130: error: field `searchbox' has incomplete type        /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:131: error: field `searchbox_title' has incomplete type  /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:132: error: field `searchbox_border' has incomplete type /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:133: error: field `position_indicator' has incomplete ty pe                                                                                                                 /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:134: error: field `menubox' has incomplete type          /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:135: error: field `menubox_border' has incomplete type   /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:136: error: field `item' has incomplete type             /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:137: error: field `item_selected' has incomplete type    /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:138: error: field `tag' has incomplete type              /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:139: error: field `tag_selected' has incomplete type     /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:140: error: field `tag_key' has incomplete type          /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:141: error: field `tag_key_selected' has incomplete type /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:142: error: field `check' has incomplete type            /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:143: error: field `check_selected' has incomplete type   /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:144: error: field `uarrow' has incomplete type           /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:145: error: field `darrow' has incomplete type           /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:200: error: parse error before '*' token                 /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:221: error: parse error before '*' token                 /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:223: error: parse error before '*' token                 /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:224: error: parse error before '*' token                 /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:225: error: parse error before '*' token                 /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:226: error: parse error before '*' token                 /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:228: error: parse error before '*' token                 /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c: In function `conf':                                               /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c:671: error: `KEY_MAX' undeclared (first use in this function)      /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c:671: error: (Each undeclared identifier is reported only once      /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c:671: error: for each function it appears in.)                      /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c: In function `show_help':                                          /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c:793: error: `stdscr' undeclared (first use in this function)       /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c: In function `conf_choice':                                        /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c:855: error: `KEY_MAX' undeclared (first use in this function)      make[1]: *** [mconf.o] Error 1                                                                                     make[1]: Leaving directory `/c/Users/John/esp/esp-idf/tools/kconfig'
--

John@DESKTOP-1FHIRUI MINGW32 ~/esp/hello_world                                                                                                        $ make menuconfig                                                                                                                                     make[1]: Entering directory `/c/Users/John/esp/esp-idf/tools/kconfig'                                                                                 /usr/bin/gcc -c  -DCURSES_LOC="" -DLOCALE -MD  /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c -o mconf.o                                   In file included from /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c:22:                                                                             /c/Users/John/esp/esp-idf/tools/kconfig/lkc.h:12:22: libintl.h: No such file or directory                                                             In file included from /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c:23:                                                                             /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:38:20: curses.h: No such file or directory                                                  In file included from /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c:23:                                                                             /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:103: error: parse error before "chtype"                                                     /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:103: warning: no semicolon at end of struct or union                                        /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:107: error: parse error before '}' token                                                    /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:117: error: field `screen' has incomplete type                                              /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:118: error: field `shadow' has incomplete type                                              /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:119: error: field `dialog' has incomplete type                                              /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:120: error: field `title' has incomplete type                                               /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:121: error: field `border' has incomplete type                                              /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:122: error: field `button_active' has incomplete type                                       /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:123: error: field `button_inactive' has incomplete type                                     /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:124: error: field `button_key_active' has incomplete type                                   /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:125: error: field `button_key_inactive' has incomplete type                                 /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:126: error: field `button_label_active' has incomplete type                                 /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:127: error: field `button_label_inactive' has incomplete type                               /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:128: error: field `inputbox' has incomplete type                                            /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:129: error: field `inputbox_border' has incomplete type                                     /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:130: error: field `searchbox' has incomplete type                                           /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:131: error: field `searchbox_title' has incomplete type                                     /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:132: error: field `searchbox_border' has incomplete type                                    /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:133: error: field `position_indicator' has incomplete type                                  /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:134: error: field `menubox' has incomplete type                                             /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:135: error: field `menubox_border' has incomplete type                                      /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:136: error: field `item' has incomplete type                                                /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:137: error: field `item_selected' has incomplete type                                       /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:138: error: field `tag' has incomplete type                                                 /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:139: error: field `tag_selected' has incomplete type                                        /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:140: error: field `tag_key' has incomplete type                                             /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:141: error: field `tag_key_selected' has incomplete type                                    /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:142: error: field `check' has incomplete type                                               /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:143: error: field `check_selected' has incomplete type                                      /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:144: error: field `uarrow' has incomplete type                                              /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:145: error: field `darrow' has incomplete type                                              /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:200: error: parse error before '*' token                                                    /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:221: error: parse error before '*' token                                                    /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:223: error: parse error before '*' token                                                    /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:224: error: parse error before '*' token                                                    /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:225: error: parse error before '*' token                                                    /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:226: error: parse error before '*' token                                                    /c/Users/John/esp/esp-idf/tools/kconfig/lxdialog/dialog.h:228: error: parse error before '*' token                                                    /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c: In function `conf':                                                                                  /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c:671: error: `KEY_MAX' undeclared (first use in this function)                                         /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c:671: error: (Each undeclared identifier is reported only once                                         /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c:671: error: for each function it appears in.)                                                         /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c: In function `show_help':                                                                             /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c:793: error: `stdscr' undeclared (first use in this function)                                          /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c: In function `conf_choice':                                                                           /c/Users/John/esp/esp-idf/tools/kconfig/mconf.c:855: error: `KEY_MAX' undeclared (first use in this function)                                         make[1]: *** [mconf.o] Error 1                                                                                                                        make[1]: Leaving directory `/c/Users/John/esp/esp-idf/tools/kconfig'
--

```

**Comment by johnr2017** [comment]
A reinstall of a new msys32 installation was the answer.
There may have been a profile setting mix up with multiple msys installations.
